hxd/res/Config.hx

```diff
@@ -67,7 +67,9 @@ class Config {
 			Unknown;
 		switch( pf ) {
 		case HL:
+			#if !heaps_enable_hl_mp3
 			ignoredExtensions.set("mp3", true);
+			#end
```

Comment:
Should mp3 exclude support exclude ogg ? The way we currently deal with it is ignore either one extension or the other, so we can build for either flash (+mp3) or hl (+ogg) and have both mp3 and ogg files in res directory. But that's only for quite old projects and we only use hl/ogg recently.

Reply:
Not particularly. I would expect this flag being used for cases when target platform (flash/html5) supports only mp3, and HL used only for development due to debugger. In such case being able to load mp3 files directly more desirable than mangling 2 formats. 
Predicting "but we can use wav and then convert them" - yes and no. For our particular case, where we use Heaps to develop game on a game jam, being able to just drop mp3 files sent by composer is more convenient than downloading wav files and having them to be converted. 

On a side note, your remarks regarding FLAC made me research flac format a bit and I have WIP flac reader for `format` library. No estimates when it'll be done thought. (Currently I'm able to read the file, but no audio decoding is present and not all metadata blocks are supported)